_topics/en/pay-to-contract-outputs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Pay-to-Contract (P2C) protocols
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: p2c
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+  - Scripts and Addresses
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: Pay to Contract Protocol
+      link: https://arxiv.org/abs/1212.3257
+
+## Optional.  Each entry requires "title" and "url".  May also use "feature:
+## true" to bold entry and "date"
+optech_mentions:
+  - title: "Taproot history: pay to contract protocol"
+    url: /en/newsletters/2021/10/20/#pay-to-contract
+    date: 1970-01-01
```

Comment:
I believe this date is part of the boilerplate new topic text that can be removed.